system/CodeIgniter.php

```diff
@@ -871,9 +847,7 @@ protected function createController()
      * CI4 supports three types of requests:
      *  1. Web: URI segments become parameters, sent to Controllers via Routes,
      *      output controlled by Headers to browser
-     *  2. Spark: accessed by CLI via the spark command, arguments are Command arguments,
-     *      sent to Commands by CommandRunner, output controlled by CLI class
```

Comment:
Since this is removed, does this mean that the `CommandRunner` will be removed? Or deprecated, at least?

Reply:
The ``CommandRunner`` class will be removed as there is no point in it. I'm already preparing changes.